tests/test_api.py

```diff
@@ -19,6 +19,22 @@ def test_glottolog_invalid_repos(tmpdir):
         Glottolog(str(tmpdir))
 
 
+def test_cache(caching_api, api):
+    lang = api.languoid('abcd1234')
+    assert lang is not api.languoid('abcd1234')
+
+    lang = caching_api.languoid('abcd1234')
+    assert lang is caching_api.languoid('abcd1234')
+    assert caching_api.languoid('aaa') is lang
```

Comment:
maybe chain comparisons? `lang is caching_api.languoid('abcd1234') is caching_api.languoid('aaa')`